pkg/resources/schema.go

```diff
@@ -213,73 +213,90 @@ func CreateContextSchema(ctx context.Context, d *schema.ResourceData, meta any)
 	database := d.Get("database").(string)
 	id := sdk.NewDatabaseObjectIdentifier(database, name)
 
-	dataRetentionTimeInDays,
-		maxDataExtensionTimeInDays,
-		externalVolume,
-		catalog,
-		replaceInvalidCharacters,
-		defaultDDLCollation,
-		storageSerializationPolicy,
-		logLevel,
-		traceLevel,
-		suspendTaskAfterNumFailures,
-		taskAutoRetryAttempts,
-		userTaskManagedInitialWarehouseSize,
-		userTaskTimeoutMs,
-		userTaskMinimumTriggerIntervalInSeconds,
-		quotedIdentifiersIgnoreCase,
-		enableConsoleOutput,
-		err := GetAllDatabaseParameters(d)
-	if err != nil {
-		return diag.FromErr(err)
-	}
+	create := true
 
-	opts := &sdk.CreateSchemaOptions{
```

Comment:
oh noooo, this will conflict with my change :(

Reply:
:(